python-threatexchange/threatexchange/cli/tests/e2e_test_helper.py

```diff
@@ -62,7 +62,7 @@ def assert_cli_output(
     def assert_cli_usage_error(
         self, args: t.Iterable[str], msg_regex: str = None
```

Comment:
nit/ignorable: should `msg_regex` be `t.Optional[str]`

Reply:
Seems like it, it used to be accepted by previous versions. Will try and grab in a followup